packages/core/src/extensions/SideMenu/SideMenuPlugin.ts

```diff
@@ -249,7 +273,16 @@ export class SideMenuView<
   onDrop = (event: DragEvent) => {
     this.editor._tiptapEditor.commands.blur();
 
+    // ProseMirror doesn't remove the dragged content if it's dropped outside
```

Comment:
hmm, this would also remove the content when there's a drop in an invalid element right? e.g.: what happens if you drop on the playground sidebar?

Reply:
Nah, it's a `drop` event so it only fires when you attempt to drop on a valid target - this would be an issue if we were using `dragend` instead.